core/src/main/java/org/opensearch/sql/expression/operator/arthmetic/MathematicalFunction.java

```diff
@@ -535,6 +492,42 @@ private static DefaultFunctionResolver truncate() {
             DOUBLE, DOUBLE, INTEGER));
   }
 
+  /**
+   * Definition of multiply(x, y) function.
+   * Returns the number x multiplied by number y
+   * The supported signature of multiply function is
+   * (x: BYTE/SHORT/INTEGER/LONG/FLOAT/DOUBLE, y: BYTE/SHORT/INTEGER/LONG/FLOAT/DOUBLE)
+   * -> wider type between types of x and y
+   */
+  private static DefaultFunctionResolver multiplyFunction() {
```

Comment:
Why `multiply` here, but all rest in ArithmeticFunction?

Reply:
That was left over from before I decided to make a file for the arithmetic functions, it has now been removed